pkgs/by-name/di/dix/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "dix";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "bloxx12";
+    repo = "dix";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-vfU5ZDf/IZfmQmv9QJUin9mgMYUT77C3rWfibIIr/Yo=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-ktnYXl1a+QBvo7Ou9nRsANauDcvEnYiH2Dyw5zO4Idw=";
+
+  meta = {
+    homepage = "https://github.com/bloxx12/dix";
+    description = "A blazingly fast tool to diff Nix related things.";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ bloxx12 ];
```

Comment:
Would you be interested in adding me here as well? If we're going to be using Dix in NH, I'd also like to make sure that the package is maintained (even though we'll be using Dix as a crate)